### DISCUSSION TITLE
Spending limits are broken

### DISCUSSION THREAD

**Discussion by damienwebdev** [community/community#46698] [Codespaces] [answered]
My org uses Codespaces, and we have a limit set to $200/mo. Recently, I got a number of reports indicating the following:

![Screenshot_20230207-232950.png](https://user-images.githubusercontent.com/9029654/217433852-e0df117a-15fd-4889-975a-2d4835204456.png)

We're nowhere near the limit ($23.17/200) and I tried doubling the limit several times up to $4000 and it never resolved. It was only resolved by disabling the limit.

What's going on?

**Comment by cbini** [comment]
Same here. Just turned off our limit in the meantime, but gonna open a ticket with support

**Comment by pdeva** [comment]
we are running into the same issue.

**Comment by keiranmraine** [comment]
I've raised a ticket as I'm not in Southeast Asia where the status page indicates issues being centred (and I really hope I'm not being routed there from UK).  They have just updated the status:

> Update - We continue to work to mitigate issues with Codespaces in Southeast Asia. We are also investigating reports of customers experiencing billing issues in other regions.
Feb 08, 2023 - 10:57 UTC

https://www.githubstatus.com/

**Comment by rneatherway** [comment, marked as answer]
👋 Hello from GitHub. We believe that this billing issue has been fixed:

> Update - We continue to work to mitigate issues with Codespaces in Southeast Asia. Problems with billing in other regions should now be resolved.
> Feb 08, 2023 - 12:40 UTC

I'd be grateful to know if it is or isn't fixed for you.

**Reply by ztl8702** [reply]
Seems fixed for me

**Comment by juanpin-humane** [comment]
This is broken again !!! 

![CleanShot 2023-03-23 at 12 01 02@2x](https://user-images.githubusercontent.com/98841829/227320818-2463d1fc-212d-4e4e-a46b-393dfac0ee85.jpg)

**Reply by aprilla** [reply]
@juanpin-humane please contact your organization administrator. A spending limit was recently added at the [Enterprise level](https://docs.github.com/en/billing/managing-billing-for-github-codespaces/managing-the-spending-limit-for-github-codespaces) restricting Codespaces spending limits of your organization.
